src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -16,55 +18,74 @@ const getWeekdays = (date, locale) => {
   return out;
 };
 
-const ALL_DATE_SEPERATORS = / |,|\.|-|\//;
+const ALL_DATE_SEPARATORS = / |,|\.|-|\//;
+
 const clearStringFromIEGeneratedCharacters = string =>
   string.replace(/[^\x00-\x7F]/g, ''); // eslint-disable-line no-control-regex
+
+const zeroFill = (number, width) => {
+  const _number = number | 0; // coerce number to integer
```

Comment:
Using underscore here is confusing, since in javascript, the underscore is used for private methods and properties. This here is neither, it's a local variable.

Reply:
Hmm, at best it's a practice by some people, not anything enforced by JS syntax/semantics.

But it's been deleted anyway by refactoring zeroFill below.